File access: tweak procedure on installing terminal and ssh add-on (#32790)
home-assistant/home-assistant.github.io: source/_includes/common-tasks/file_access.md

---

### Installing and using the SSH add-on

If you want to use the Home Assistant command line or an SSH client, you can do this through the **Terminal & SSH** add-on.

The **Terminal & SSH** add-on provides the following functionalities:

- It provides a web terminal that you can access from the Home Assistant user interface.
- It allows you to use the Home Assistant Command Line Interface (CLI) which provides custom commands for checking logs, stopping and starting Home Assistant and add-ons, creating/restoring backups, and more.
  - For a list of command line commands, refer to [Home Assistant via Command Line](/common-tasks/os#home-assistant-via-the-command-line).
- It allows connecting to your system using an SSH client.
- It also includes common tools like nano and vi editors.

> - The Terminal & SSH add-on does **not provide** access to the underlying host file system. For access to the host file system, refer to one of the other methods listed in this section, such as [using the Samba add-on](#installing-and-using-the-samba-add-on).

Reviewer: ```suggestion
- The Terminal & SSH add-on does **not provide** access to the underlying host file system.
```
For that you would need to setup [debugging](https://developers.home-assistant.io/docs/operating-system/debugging) which is probably not something we would want to add here.

---

### Installing and using the SSH add-on

If you want to use the Home Assistant command line or an SSH client, you can do this through the **Terminal & SSH** add-on.

The **Terminal & SSH** add-on provides the following functionalities:

- It provides a web terminal that you can access from the Home Assistant user interface.
- It allows you to use the Home Assistant Command Line Interface (CLI) which provides custom commands for checking logs, stopping and starting Home Assistant and add-ons, creating/restoring backups, and more.

>   - For a list of command line commands, refer to [Home Assistant via Command Line](https://www.home-assistant.io/hassio/commandline/).

Reviewer: ```suggestion
  - For a list of command line commands, refer to [Home Assistant via Command Line](/common-tasks/os#home-assistant-via-the-command-line).
```

- It allows connecting to your system using an SSH client.

> - It also includes nano and vi editors.

Reviewer: ```suggestion
- It also includes common tools like nano and vi editors.
```
🤷‍♂️

> Note: The Terminal & SSH add-on does *not* provide access to the underlying host file system.

Reviewer: ```suggestion
<div class="note">

    The Terminal & SSH add-on does *not* provide access to the underlying host file system.

</div>
```
Maybe wrap this in a note div?

Author: Integrated it into the list instead.

To get started with the **Terminal & SSH** add-on, follow these steps:

> 1. In the bottom left, select your user to open the {% my profile title="**Profile**" %} page. Make sure **Advanced Mode** is activated.

Reviewer: ```suggestion
1. In the bottom left, select your user to open the {% my profile title="**Profile**" %} page. Make sure **Advanced Mode** is enabled.
```
Looks like we are mostly using "enabled" for this.

2. To install the add-on, go to the add-on store under [**Settings** > **Add-ons**](https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_ssh) and install the **Terminal & SSH** add-on.
3. To use the web terminal, **start** the add-on, then select **Open Web UI**.

>    - You can now start typing your [commands](https://www.home-assistant.io/hassio/commandline/).

Reviewer: ```suggestion
   - You can now start typing your [commands](/common-tasks/os#home-assistant-via-the-command-line).
```

4. If you want to access from an ssh client, you need to enter credentials:
   - Open the **Configuration** page.

>    - Enter a password or public key.

Reviewer: ```suggestion
   - Enter a password or authorized Keys.
```
ref: https://github.com/home-assistant/addons/blob/master/ssh/translations/en.yaml#L4
